test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.cs

```diff
@@ -289,16 +289,16 @@ public void CannotStartConnectionWithInvalidTransportType(TransportType requeste
 
             var connection = new HttpConnection(new Uri("http://fakeuri.org/"), new TestTransportFactory(mockTransport.Object), loggerFactory: null, httpMessageHandler: mockHttpHandler.Object);
 
-            var receivedInvoked = false;
-            connection.Received += m =>
+            var onReceivedInvoked = false;
+            connection.OnReceived((_, __) =>
             {
-                receivedInvoked = true;
+                onReceivedInvoked = true;
                 return Task.CompletedTask;
-            };
+            }, null);
```

Comment:
super nit: pass `onReceivedInvoked` as the state
And for other tests

Reply:
Talked offline. Doesn't apply for this test